backend/models/product.model.js

```diff
@@ -14,27 +14,27 @@ const reviewSchema = mongoose.Schema(
   { timestamps: true }
 );
 
+
 const productSchema = mongoose.Schema(
   {
-    productId: { type: String, unique: true, required: true },
     updatedBy: {
       type: mongoose.Schema.Types.ObjectId,
       required: true,
       ref: "Staff",
     },
-    name: {
+    title: {
       type: String,
       required: true,
+      unique: true
     },
-    productImage: {
+    image: {
+      type: String,
+      required: true,
+    },
+    category: {
       type: String,
       required: true,
     },
```

Comment:
I also added some new fields for the product models, Will resolve the conflict when merged